### DISCUSSION TITLE
KoFI Integration?

### DISCUSSION THREAD

**Discussion by Decollates** [gameserverapp/Platform#686] [Suggestions]
Would it be possible to integrate KoFi into GSA, same as Patreon? 

https://help.ko-fi.com/hc/en-us/articles/360004162298-Does-Ko-fi-Have-an-API-or-Webhook-#i-want-to-use-the-ko-fi-api-for-my-page-0-1

**Comment by safestring** [comment]
Plus 1

**Comment by mvaessen** [comment]
I checked on this, and unfortunately is there a lot of stuff missing to make this work.

For example, there is no OAuth integration, allowing people to connect their Ko-Fi account. OAuth is basically the social login
And we're also missing API endpoints to fetch which memberships there are. 

Basically, they only have webhooks. And webhooks are one-way communication (from Ko-Fi to GSA).

If they had an API, we could perhaps make it work. But without OAuth __and__ an API, is it impossible to make this work.

**Reply by Decollates** [reply]
Bummer! Thank you for looking into this.
